src/format.iced

```diff
@@ -0,0 +1,122 @@
+stream = require('keybase-chunk-stream')
+util = require('./util')
+
+space = Buffer.from(' ')
+newline = Buffer.from('\n')
+punctuation = Buffer.from('.')
+
+words_per_line = 200
+chars_per_word = 15
+
+noop = () ->
+
+exports.FormatStream = class FormatStream extends stream.ChunkStream
+
+  __format : (chunk) ->
+    return Buffer.concat([space, chunk])
+
+  _format : (chunk) ->
+    if chunk.length < chars_per_word then return chunk
+    results = []
+    for i in [0...chunk.length] by chars_per_word
+      word = chunk[i...i+chars_per_word]
+      if i+chars_per_word >= chunk.length
+        results.push(word)
```

Comment:
Same concern about partial words as above, this time for chunks that aren't multiples of `chars_per_word`.